master/internal/experiment_filter.go

```diff
@@ -132,6 +132,40 @@ func expColumnNameToSQL(columnName string) (string, error) {
 	return col, nil
 }
 
+func runColumnNameToSQL(columnName string) (string, error) {
+	// To prevent SQL injection this function should never
+	// return a user generated field name
+
+	filterExperimentColMap := map[string]string{
+		"id":                   "e.id",
+		"description":          "e.config->>'description'",
```

Comment:
Using LEFT JOIN for this so that runs without it will just have NULL. I believe frontend and simply block actions that require this data if the experiment id field is NULL. @ashtonG could this work just like what fontend does with 'unmanaged' experiments

Reply:
yes for individual actions, but no on bulk actions. in situations where the user is operating off of a "select all" selection, the runs in the set will possibly include runs that haven't been paginated in yet, so we wouldn't know if the experiment id field is null or not.